.github/workflows/validate-linked-work-item.yaml

```diff
@@ -0,0 +1,19 @@
+name: 'Validate linked work item'
+
+on: 
+  pull_request:
+    types: [opened, reopened, edited]
+    branches:
+      - main 
+
+permissions:
+    pull-requests: write
+    issues: write

```

Comment:
can we disable it for issues for now?

Reply:
No because the issue api is used to write comments